src/controller/CHIPDeviceController.h

```diff
@@ -367,6 +375,32 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public Rendezvous
      */
     CHIP_ERROR CloseBleConnection();
 #endif
+    /**
+     * @brief
+     *   Discover devices advertising as commissionable that match the long discriminator.
+     * @return CHIP_ERROR   The return status
+     */
+    CHIP_ERROR DiscoverCommissioningLongDiscriminator(uint16_t long_discriminator);
```

Comment:
Short discriminator matching is also needed.

Suggest adding API for DiscoverCommissioningShortDiscriminator as well, which will be used for numerical pairing code

Reply:
There are actually lot of them missing now - there were new ones added in the new ballot. If it's ok, I'd like to add those in another PR - this PR keeps getting clobbered by merge problems and I'd like to get the base in then fill in remaining _sub and TXT parsing in a smaller change.

Reply:
Sure, you can open an issue for the short discriminator support (in fact I think I saw one)

Reply:
There are also subtypes for _T (device type), _C (commissioning mode), _A (additional pairing mode). Please be sure to capture all of these as TODO, thanks.